### PR TITLE
Automatically start transactions from incoming trace in ASP.NET Core

### PR DESCRIPTION
**Pull request by Tyrrrz**

### CODE REVIEW

File: src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs [hunk 1]
```diff
@@ -56,5 +56,23 @@ internal static class HttpContextExtensions
...
+                {
+                    return null;
+                }
+
+                return SentryTraceHeader.Parse(value.ToString());
+            }
+            catch
+            {
+                // Parsing can fail
+                return null;
```
Comment (bruno-garcia) [inline src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs:74]: No logger.logerror? We could be hiding a bug here

Comment (Tyrrrz) [reply src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs:74]: @bruno-garcia updated

**Review by bruno-garcia — APPROVED** [review]
🚢 

:shipit:

**Comment by lucas-zimerman** [comment]
It's failing on macos

**Comment by bruno-garcia** [comment]
flaky tests still lives

**Pull request merged**